flow/SimpleOpt.h

```diff
@@ -280,7 +280,10 @@ enum _ESOFlags {
 	SO_O_ICASE_WORD = 0x0400,
 
 	/*! Case-insensitive comparisons for all arg types */
-	SO_O_ICASE = 0x0700
+	SO_O_ICASE = 0x0700,
+
+	/*! Case-insensitive comparisons for all hyphens and underscores after leading hyphens */
+	SO_O_ICASE_HYPHEN_AND_UNDERSCORE = 0x1000
```

Comment:
This isn't technically case-insensitive, so the `ICASE` label doesn't really apply.

Reply:
Do you think: 
SO_O_HYPHEN_TO_UNDERSCORE
is a good name here? Thanks!

Reply:
Yeah, that sounds good